v4/export/config.go

```diff
@@ -117,6 +129,53 @@ func (conf *Config) GetDSN(db string) string {
 	return dsn
 }
 
+func (config *Config) ExternalStorage(ctx context.Context) storage.ExternalStorage {
+	config.externalStorageOnce.Do(func() {
+		s, err := config.createExternalStorage(ctx)
+		if err != nil {
+			panic(err)
```

Comment:
We'd better return the error to the outside function instead of `panic`. Our users may get confused if they saw panic info.